src/ontology/mondo-edit.obo

```diff
@@ -274335,6 +274336,7 @@ relationship: has_characteristic MONDO:0021140 {source="https://orcid.org/0000-0
 relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/95 {source="MONDO:mim2gene_medgen"} ! SLC33A1
 property_value: confidence "2.692307692307692" xsd:double
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/4069" xsd:anyURI
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/6765#issuecomment-1797800311" xsd:anyURI
```

Comment:
should this then be https://github.com/monarch-initiative/mondo/issues/6849?

also, have references to a specific comment in an issue been used as the link before?

Reply:
I think it's fine to reference the comment. I added that because it has more context.